justfile

```diff
@@ -1,42 +1,47 @@
 set dotenv-load := false
 
+# Show all available recipes
+default:
+  @just --list --unsorted
```

Comment:
this is quite helpful!

Reply:
Agreed, @dhruvkb was the one who added it in the other PR!